image_publisher/CMakeLists.txt

```diff
@@ -44,24 +44,28 @@ ament_target_dependencies(image_publisher
 rclcpp_components_register_nodes(image_publisher "${PROJECT_NAME}::ImagePublisher")
 set(node_plugins "${node_plugins}${PROJECT_NAME}::ImagePublisher;$<TARGET_FILE:ImagePublisher>\n")
 
-install(DIRECTORY include/${PROJECT_NAME}/
-    DESTINATION include/${PROJECT_NAME}
-    FILES_MATCHING PATTERN "*.h")
-
 add_executable(image_publisher_node src/image_publisher_node.cpp)
 target_link_libraries(image_publisher_node ${ament_LIBRARIES} image_publisher)
 ament_target_dependencies(image_publisher_node
   ${DEPENDENCIES})
 
-install(TARGETS image_publisher_node
-        DESTINATION lib/${PROJECT_NAME}
+install(
+  DIRECTORY include/
+  DESTINATION include
 )
-install(TARGETS image_publisher
-        DESTINATION lib/${PROJECT_NAME}
-)
-install(TARGETS image_publisher
-        DESTINATION bin
+
+install(
+  TARGETS image_publisher
+  EXPORT export_${PROJECT_NAME}
```

Comment:
The `EXPORT` portion shouldn't be necessary. It is usually paired with `ament_export_targets()` (previously `ament_export_interfaces()`) but that isn't required here.